services/src/test/java/io/scalecube/services/ServiceRegistryImplTest.java

```diff
@@ -0,0 +1,54 @@
+package io.scalecube.services;
+
+import static org.junit.Assert.assertEquals;
+
+import io.scalecube.cluster.Cluster;
+import io.scalecube.services.annotations.AnnotationServiceProcessor;
+import io.scalecube.services.annotations.ServiceProcessor;
+import io.scalecube.transport.Transport;
+
+import org.junit.Test;
+
+public class ServiceRegistryImplTest {
+
+  @Test
+  public void test_service_registry() {
+
+    Cluster cluster = Cluster.joinAwait();
+    Transport transport = Transport.bindAwait();
+
+    ServicesConfig services = ServicesConfig.empty();
+    ServiceProcessor serviceProcessor = new AnnotationServiceProcessor();
+
+    ServiceCommunicator sender = new ClusterSender(cluster);
+    
+    ServiceRegistryImpl registry = new ServiceRegistryImpl(cluster, sender, services, serviceProcessor);
```

Comment:
What this test is checking? I mean is it like the constructor didn't throw any exception? I am not sure that such kind of test is really needed... since if you actually use this class anywhere it obviously will be covered by other test scenarios.